src/plugins/vis_type_timeseries/server/lib/search_strategies/strategies/abstract_search_strategy.ts

```diff
@@ -56,12 +56,11 @@ export class AbstractSearchStrategy {
   }
 
   async search(req: ReqFacade, bodies: any[], options = {}) {
-    const [, deps] = await req.framework.core.getStartServices();
     const requests: any[] = [];
     bodies.forEach((body) => {
       requests.push(
-        deps.data.search
-          .search(
+        req.requestContext
+          .search!.search(
```

Comment:
can we do something with code to don't add search`!`  in each place? Looks like something wrong with types, search should not be optional

Reply:
i think it's a problem with the context TS again :(